### ISSUE TITLE
An option to remember which images have been compressed

### ISSUE THREAD

**Issue by macemmek** [websperts/tinypng-cli#10]
Such option could save TinyPNG hits. The information could be stored in a json file. (Like [this](https://github.com/marrone/grunt-tinypng) Grunt task does.)

**Comment by artiswilliams** [comment]
Agreed.  I used that grunt task previous to switching to all npm scripts.  Would appreciate this option.  Uses MD5 hash of the file compared against a json file of md5 hashes of previously compressed files.
